docs/rules/DAP245.md

```diff
@@ -0,0 +1,22 @@
+﻿# DAP245
+
+There is a non-zero chance of Dapper treating character sequence in SQL query like `? something ?` as a pseudo-positional parameter,
```

Comment:
not sure `? something ?`  will get treated - worth checking, but I think it would need to be `?something?` - and we want to call out that it is in a literal, i.e. `'blah ?something blah'`; also worth checking the spaces inside the match - i.e. `?thisismystringdata?` vs `?this is my string data?`

Reply:
spaces in literal between questions marks will not be considered a pseudo-positional.
I have rewritten the docs accordingly.

Reply:
Also mentioned that it is **a literal** explicitly